src/ops/ops.go

```diff
@@ -1260,3 +1260,94 @@ func (o *ops) OverwriteOsImage(osImage, device string, extraArgs []string) error
 	}
 	return nil
 }
+
+type DiskOps struct {
```

Comment:
why not to create new file for diskops?

Reply:
Because in order to use diskOps you  need ops anyway so if you want to use it outside of assisted-installer you will need to import both packages